_posts/2022-08-04-5.6.0-released.md

```diff
@@ -66,7 +66,7 @@ actually started in 5.5.0, but didn't make that release for various reasons.
 
 In order to enable dynamic shadows, a mod or game must opt-in to it. You can use
 the [Enable Shadows](https://content.minetest.net/packages/ROllerozxa/enable_shadows/)
-mod to enable shadows in any game.
+mod to force them to be enabled in any game but be aware this may not work as expected.
```

Comment:
```suggestion
mod to allow them to be enabled in any game but be aware this may not work as expected.
```

it doesn't force them on, the user still needs to check the setting